ansible/roles/runner/templates/config.yml

```diff
@@ -5,6 +5,7 @@ credentials:
     token: {{ github_token }}
 tasks_file: .freeipa-pr-ci.yaml
 whitelist_file: /root/freeipa-pr-ci/whitelist.yml
+no_task_backoff_time: {{ task_backoff_time | default(300) }}
```

Comment:
Please put the default into `runner/roles/defaults/main.yml` so it's easier to find out this is a variable you can modify for the role.